notebooks/Notebook2.md

```diff
@@ -18,17 +20,42 @@ exciting milestones, preliminary results, etc. Feel free to include images
+
+Beyond that, as mentioned, I also created a spreadsheet which specifies the three distinct types, and the functionality associated with each one.
+
+![image](https://user-images.githubusercontent.com/79538073/230794237-7f391bd4-6353-43d8-aa84-9240bcaecdcb.png)
+
+{screenshot of a portion of the spreadsheet! The sheet guided the aforementioned programming progress this week}
+
 ## Questions
 
 **What is the most pressing issue for your project? What design decision do
 you need to make, what implementation issue are you trying to solve, or how
 are you evaluating your design and implementation?**
 
+My largest issues are design. I knew design would be a bigger component of the project, and I need to make the language actually usable instead of what is now - just a series of implementations. 
+
+Of course, I still have to add many more operators and functions and flow control structures, but I do think I have that aspect under control. Getting the design down is a more pressing issue now.
+
+To move forward with design, I will be talking to my Linguistics Prof again - next week. Discuss what might be intuitive, and what might not. Then, I will be talking to Prof Ben about HOW to actually mold the current design to match the desired one. A lot of it, I feel like I know how to do or I can figure out myself. But some of it, such as the "/text here/" problem specied in my previous notebook entry, are harder and I may need some help. 
+
 **What questions do you have for your critique partners? How can they best help
 you?**
 
+Some specific design ideas may require parsers and shifting(?) towards an external DSL. I was wondering if that might be the right direction.
+
+And as last week, what are some forms of String manupulation that you would like to see? What are some operators I could add?
+
 **How much time did you spend on the project this week? If you're working in a
 team, how did you share the work?**
 
+4 hours on coding and implementation
+3 hours on ideating, and creating a spreadsheet and coming up a proper, organised list of what I want to do. 
+50 minutes on filling on this notebook entry, and bringing all my material together. 
+70 minutes on critiques (in class, and outside).
+
 **Compared to what you wrote in your contract about what you want to get out of this
 project, how did this week go?**
+
+Pretty great! I wanted to have a few working datatypes with implicit conversions(in and out), operators, functions, flow control, etc. - and I got that! I need to now bend Scala a bit to get the desired design, or implement a parser for certain design ideas. 
```

Comment:
Looking at your code so far, it looks like you have made a lot of progress on your implementation. I am wondering if an error is thrown if you try to subtract characters from an unsorted or phenomic string that are not already in the string. I am also curious if a regular strings can be converted to phenomic strings, or if there would even be a reson to do that. For example, would "hello" equal "<hE:l@>"? The example code you gave is also really helpful to understand how the language will actually be used. The way you set up your case classes is very efficient, and I hope to use this as an example when designing my own language, since I will also be creating classes.